WordPress/src/main/java/org/xmlrpc/android/XMLRPCUtils.java

```diff
@@ -289,6 +292,14 @@ private static String verifyXmlrpcUrl(final String siteUrl, final String httpUse
                     // Endpoint found and works fine.
                     return url;
                 }
+            } catch (XMLRPCUtilsException e) {
+                if (e.kind == XMLRPCUtilsException.Kind.ERRONEOUS_SSL_CERTIFICATE ||
+                    e.kind == XMLRPCUtilsException.Kind.HTTP_AUTH_REQUIRED ||
+                    e.kind == XMLRPCUtilsException.Kind.MISSING_XMLRPC_METHOD) {
```

Comment:
> Why not let the loop continue with the rest of the URLs even though it will probably lead to the same error for all URLs?

Yes, we will get the same exception.